haskell/ghc_bindist.bzl

```diff
@@ -130,13 +130,17 @@ haskell_toolchain(
     tools = "{tools}",
     version = "{version}",
     compiler_flags = {compiler_flags},
+    haddock_flags = {haddock_flags},
+    repl_ghci_args = {repl_ghci_args},
     exec_compatible_with = {exec_constraints},
     target_compatible_with = {target_constraints},
 )
         """.format(
             tools = "@{}//:bin".format(ctx.attr.bindist_name),
             version = ctx.attr.version,
             compiler_flags = ctx.attr.compiler_flags,
+            haddock_flags = str(ctx.attr.haddock_flags),
```

Comment:
`str()` is unnecessary here. Notice how you're not using it in the line just above. In fact it's likewise unnecessary in the corresponding code in `nixpkgs.bzl` (but that's not a concern for this commit).

Reply:
Fixed